### PR TITLE
feat:  Added Auth

### PR DESCRIPTION
**Pull request by jalajcodes**
Added Authentication
Files to Review:

```
frontend/src/Components/Header.js
frontend/src/Pages/Auth.js
frontend/src/Reducers/authReducer.js
frontend/src/Contexts/authContext.js
```

Live: https://max-store-app.herokuapp.com/auth

### CODE REVIEW

File: frontend/src/Pages/Auth.js [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+      "register",
+      signupData.email,
+      signupData.password,
+      `${signupData.firstName} ${signupData.lastName}`
+    );
+  };
+
+  useEffect(() => {
+    const handleTabChange = () => {
+      console.log("att");
```
Comment (abhisheksahani-nova) [inline frontend/src/Pages/Auth.js]: remove console.log after testing

File: frontend/src/Pages/Auth.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+      const tabs = document.querySelectorAll(".tabs h3 a");
```
Comment (abhisheksahani-nova) [inline frontend/src/Pages/Auth.js:55]: Avoid using querySelector as said by tanay, as best practice

Comment (jalajcodes) [reply frontend/src/Pages/Auth.js:55]: Yes its already in my todo list. Just focusing on completing the features now, will refactor the code later on.

File: frontend/src/Pages/Auth.js (hunk 1 above)
Comment (jalajcodes) [reply frontend/src/Pages/Auth.js]: thanks for pointing out.

**Review by ApurvaSawant11 — APPROVED** [review]
Everything is looking great. Authentication is also working properly. :)

**Pull request merged**